### PR TITLE
Reupdate all incompressible flow examples

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- Some incompressible flow examples did not run anymore because of parameter file errors. 

# Description of the solution

- I reran all of them, updated them, made sure everything is all good. I took the opportunity to improve some of them at the same time. It is really just maintenance work, but it is necessary

# How Has This Been Tested?

- reran all examples and checked their results

# Documentation

- Also updated documentation

# Future changes

- I will do the same thing for all the other examples folder

# Comments

### CODE REVIEW

**Review by lpsaavedra — CHANGES_REQUESTED** [review]
Always good to keep the examples up to date. I have only one comment in the Taylor-Couette flow using Nitsche IB example

File: doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst [hunk 1]
```diff
@@ -107,8 +107,8 @@ Gauss quadrature points of the immersed mesh to represent the immersed body. For
       end
     end
 
-The ``beta`` coefficient is a parameter used to enforce the Nitsche IB. It's value is generally between 1 and 100, according to the size of the mesh. 
-A value of 10 is reasonable. Then, we specify the ``number of solids`` geometries that with be represented with Nitsche IB. 
+The ``beta`` coefficient is a parameter used to enforce the Nitsche IB. It's value is generally between 1 and 100 (10 being a reasonable value). The restriction automatically scales with the mesh size.
```
Comment (voferreira) [inline doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst:110]: I think you could refer to Jeanne's paper here.

**Review by voferreira — COMMENTED** [review]
Very good to update them indeed. Apart from the suggestion on Nitsche, I suggest that examples in the documentation and examples folders have the same name. For instance, the flow around Ahmed's body example is "2d-transient-around-ahmed-body/" in the doc and "2d-ahmed-body/" and "2d-ahmed-body-simples/" in the example. I am testing the .prm files now to see if everything works as expected before approving.

File: doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst (hunk 1 above)
Comment (blaisb) [reply doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst]: Well the paper does not really bring anything to the discussion here, so I would rather be succinct and just provide a value.

**Review by lpsaavedra — APPROVED** [review]

**Review by voferreira — APPROVED** [review]
All files were tested. Everything is working. From my side, it is good to go.

**Pull request merged**